dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -256,9 +262,49 @@ private WorkflowVersion updateDBVersionSourceFilesWithRemoteVersionSourceFiles(W
         for (Validation versionValidation : remoteVersion.getValidations()) {
             existingVersion.addOrUpdateValidation(versionValidation);
         }
+
+        // Setup CheckUrl
+        if (checkUrlLambdaUrl != null) {
+            publicAccessibleUrls(existingVersion, checkUrlLambdaUrl);
+        }
+
         return existingVersion;
     }
 
+    /**
+     * Sets the publicly accessible URL version metadata.
+     * If all test parameter files have publicly accessible URLs (true), then version metadata is true
```

Comment:
Isn't the goal to know if there is at least one test parameter file that has all public urls, rather than whether all test parameter files have public urls? I thought the idea was to know if there was at least one test parameter file with public urls as a best practice.

Reply:
Fair point, I think at least one makes more sense than all.

Reply:
i personally didn't interpret it that way. I would find it odd if a workflow has 99 test parameter files that aren't publicly accessible but has 1 that is and we label that with a checkmark indicating that it's following best practices and others should follow suit